src/StickPortal.js

```diff
@@ -192,9 +192,33 @@ function calculateLeft(
     return result - hostLeft
   }
 
+  if (scrollHost) {
+    return result + scrollX(nodeRef) - scrollHost.scrollLeft
+  }
+
   return result + scrollX(nodeRef)
 }
 
+function getScrollParent(element) {
+  if (!element) {
+    return null
+  }
+
+  if (element.nodeName === 'BODY' || element.nodeName === 'HTML') {
+    return null
+  }
+
+  const style = getComputedStyle(element)
+
+  if (style.overflowX === 'auto') {
```

Comment:
maybe also handle `overflow: scroll`

Reply:
Ah yeah... I somehow threw that away after I've typed it.